README.md

```diff
@@ -14,30 +14,26 @@ To build using the Apache CXF project tools use
 
 `mvn clean install -Djax-ws-cxf`
 
-Before connecting to a Windows server, you will need to ensure that the server is accessible and has been configured to allow
-unencrypted WinRM connections over http. The following batch script will configure WinRM and open port 5985 (the default WinRM
-port) on the local firewall.
-
-``` bat
-winrm quickconfig -q
-winrm set winrm/config/service/auth @{Basic="true"}
-winrm set winrm/config/service/auth @{CredSSP="true"}
-winrm set winrm/config/client/auth @{CredSSP="true"}
-winrm set winrm/config/client @{AllowUnencrypted="true"}
-winrm set winrm/config/service @{AllowUnencrypted="true"}
-winrm set winrm/config/winrs @{MaxConcurrentUsers="100"}
-winrm set winrm/config/winrs @{MaxMemoryPerShellMB="0"}
-winrm set winrm/config/winrs @{MaxProcessesPerShell="0"}
-winrm set winrm/config/winrs @{MaxShellsPerUser="0"}
-netsh advfirewall firewall add rule name=RDP dir=in protocol=tcp localport=3389 action=allow profile=any
-netsh advfirewall firewall add rule name=WinRM dir=in protocol=tcp localport=5985 action=allow profile=any
+
+### Maven dependency
+
+Add the following to your `pom.xml`:
+
 ```
+<dependency>
+  <groupId>io.cloudsoft.windows</groupId>
+  <artifactId>winrm4j</artifactId>
+  <version>0.3.2</version> <!-- WINRM4J_VERSION -->
+</dependency>
+```
+
+### Java client usage
 
 To use winrm4j in Java code, you first create a `WinRmTool` object via the static `connect` method. It exposes the methods
 `executeScript` and `executePs`, which can be used to execute batch or PowerShell statements respectively.
 
 ``` java
-WinRMTool winrm = WinRmTool.connect("my.windows.server.com", "Administrator", "password");
+WinRMTool winrm = WinRmTool.connect("my.windows.server.com", "Administrator", "pa55w0rd!");
```

Comment:
Better advice to use:

    WinRmTool.Builder builder = WinRmTool.Builder.builder("my.windows.server.com", "Administrator", "pa55w0rd!");
    builder.setAuthenticationScheme(AuthSchemes.NTLM);
    builder.port(5985);
    builder.useHttps(false);
    WinRmTool tool =  builder.build();